app/code/Magento/Captcha/Test/Unit/Observer/CheckUserForgotPasswordBackendObserverTest.php

```diff
@@ -0,0 +1,188 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+declare(strict_types=1);
+
+namespace Magento\Captcha\Test\Unit\Observer;
+
+use Magento\Captcha\Helper\Data as DataHelper;
+use Magento\Captcha\Model\CaptchaInterface;
+use Magento\Captcha\Observer\CaptchaStringResolver;
+use Magento\Captcha\Observer\CheckUserForgotPasswordBackendObserver;
+use Magento\Framework\App\Action\Action;
+use Magento\Framework\App\ActionFlag;
+use Magento\Framework\App\Request\Http as HttpRequest;
+use Magento\Framework\App\Response\Http as HttpResponse;
+use Magento\Framework\Event\Observer;
+use Magento\Framework\Message\ManagerInterface;
+use Magento\Framework\Session\SessionManagerInterface;
+use Magento\Framework\TestFramework\Unit\Helper\ObjectManager as ObjectManagerHelper;
+use PHPUnit\Framework\MockObject\MockObject;
+use PHPUnit\Framework\TestCase;
+
+/**
+ * Unit Test for \Magento\Captcha\Observer\CheckUserForgotPasswordBackendObserver
+ */
+class CheckUserForgotPasswordBackendObserverTest extends TestCase
+{
+    /**
+     * @var MockObject|DataHelper
+     */
+    private $helperMock;
+
+    /**
+     * @var MockObject|CaptchaStringResolver
+     */
+    private $captchaStringResolverMock;
+
+    /**
+     * @var MockObject|SessionManagerInterface
+     */
+    private $sessionMock;
+
+    /**
+     * @var MockObject|ActionFlag
+     */
+    private $actionFlagMock;
+
+    /**
+     * @var MockObject|ManagerInterface
+     */
+    private $messageManagerMock;
+
+    /**
+     * @var CheckUserForgotPasswordBackendObserver
+     */
+    private $observer;
+
+    /**
+     * @var MockObject
```

Comment:
Please set the `CaptchaInterface| MockObject` type here.

Reply:
thanks, updated